docs/input.md

```diff
@@ -104,6 +106,16 @@ add styling to error message (optional)
 
 ---
 
+### `errorProps`
+
+provide all props from react-native Text component
```

Comment:
May want to be more descriptive here.
Like: `Allows custom props you can pass to the error text`